apiserver/command/cmd.go

```diff
@@ -0,0 +1,81 @@
+package command
+
+import (
+	"errors"
+	"github.com/go-logr/logr"
+	"github.com/urfave/cli/v2"
+	appcatv1 "github.com/vshn/appcat-apiserver/apis/appcat/v1"
+	"github.com/vshn/appcat-apiserver/apiserver/appcat"
+	"github.com/vshn/appcat-apiserver/apiserver/vshn/postgres"
+	"sigs.k8s.io/apiserver-runtime/pkg/builder"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+type apiServerCommand struct {
+	appcatEnabled      bool
+	vshnBackupsEnabled bool
+}
+
+func Command() *cli.Command {
+	cmd := &apiServerCommand{}
+	return &cli.Command{
+		Name:   "apiserver",
+		Usage:  "Start apiserver",
+		Action: cmd.execute,
+		Flags: []cli.Flag{
+			&cli.BoolFlag{
+				Name:        "appcat-handler",
+				Usage:       "Enables AppCat handler for this Api Server",
+				Value:       true,
+				EnvVars:     []string{"APPCAT_HANDLER_ENABLED"},
+				Destination: &cmd.appcatEnabled,
+			},
+			&cli.BoolFlag{
+				Name:        "postgres-backup-handler",
+				Usage:       "Enables PostgreSQL backup handler for this Api Server",
+				Value:       false,
+				EnvVars:     []string{"VSHN_POSTGRES_BACKUP_HANDLER_ENABLED"},
+				Destination: &cmd.vshnBackupsEnabled,
+			},
+		},
+	}
```

Comment:
Are you sure the mix of `urfave/cli` and cobra actually works? I failed to get the apiserver running at all as the command requires some flags that cannot be set because `urfave/cli` doesn't recognize them.

Reply:
Yeah I just tried to deploy it and it does not work I will have to check further what is wrong